a/.babelrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
what is this `a/` directory?

Reply:
It contains the master branch files, I had to do that because quasar has its own folder structure already and I need to keep some of the assets close. When the upgrade is completed the **a** folder will be removed.

Reply:
ok then.